extensions/wikia/AdEngine3/src/slots.js

```diff
@@ -212,6 +214,13 @@ export default {
 		});
 	},
 
+	setupSizes() {
```

Comment:
This function name is not really specific but currently I don't have a better idea for it :/